File: app/src/main/java/com/widiarifki/screentest/presentation/menu/IMenuView.java
package com.widiarifki.screentest.presentation.menu;

/**
 * Created by widiarifki on 27/02/2018.
 */

public interface IMenuView {
    void writeUserName(String username);
    void goToEvent();
    void goToGuest();
    void writeEvent(String eventName);
    void writeGuest(String guestName);
}

File: app/src/main/java/com/widiarifki/screentest/presentation/menu/IMenuView.kt
package com.widiarifki.screentest.presentation.menu

/**
 * Created by widiarifki on 27/02/2018.
 */

interface IMenuView {
    fun writeUserName(username: String?)
    fun goToEvent()
    fun goToGuest()
    fun writeEvent(eventName: String?)
    fun writeGuest(guestName: String?)
}
